src/puyapy_mocks/__init__.py

```diff
@@ -0,0 +1,22 @@
+# ruff: noqa
+# types are in the correct dependency order
+
+from puyapy_mocks._primatives import *
+from puyapy_mocks._reference import *
+from puyapy_mocks._contract import *
+from puyapy_mocks._hints import *
+from puyapy_mocks._util import *
+from puyapy_mocks import (
+    arc4 as arc4,
```

Comment:
given that aliased name is identical, wouldn't

```py
from puyapy_mocks import (
    arc4,
    gtxn,
    itxn,
    op,
)
```

aid same result?

Reply:
Yep, it was to appease the ruff formatter from detecting that the import wasn't used and deleting it. I've since set noqa as the import order is important, so not really needed.